### PR TITLE
Class 4: installation and getting started

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: content/install.md [hunk 1]
```diff
@@ -1,93 +1,11 @@
...
 
-## Requirements
+Wazo Platform is an Open Source project allowing to build carrier grade programmable IP communication infrastructures. You can pick and choose the components you need to build your infrastructures with class 5 features like audio and video calls, chat, call centers, conferences, voicemail, etc., and class 4 features like security, routing, load balancing, etc. You can augment the platform and integrate it with your subscription and billing capabilities or any value-added services like Speech to Text, A.I. or sentiment analysis. These communication infrastructures can be deployed and scaled on bare metal, virtual machines or containers.
 
-- Memory: 2 GiB of memory is a tight minimum
-- Storage: 2.5 GiB of storage is a very tight minimum, 8 GiB is comfortable
+## Unified Communication
 
-## Procedure
+Follow [our guide to install the Wazo Platform Unified Communication engine](https://wazo-platform.org/install/unified-communication).
```
Comment (fredericlepied) [inline content/install.md]: Please use a relative link to allow to work locally and in CI.

Also we should not add anything after Wazo Platform to be seen as a sub-product. We should describe it as a use case of Wazo Platform like "our guide to install the Unified Communication all-in-one use case".

File: content/install.md [hunk 2]
```diff
@@ -1,93 +1,11 @@
...
-
-Note: You should replace the following values:
-* `my-token` with the authentication token
-
-```ShellSession
-$ curl -k -X GET -H 'X-Auth-Token: <my-token>' -H 'Content-Type: application/json' -d '{"firstname": "user1"}' https://wazo.example.com/api/confd/1.1/users
-```
-
-Note: the token that you have now only has permissions for configuration REST API (wazo-confd).
+Follow [our guide to install the Wazo Platform Class 4](https://wazo-platform.org/install/class-4).
```
Comment (fredericlepied) [inline content/install.md]: Same remarks:

Please use a relative link to allow to work locally and in CI.

Also we should not add anything after Wazo Platform to be seen as a sub-product. We should describe it as a use case of Wazo Platform like "our guide to install the Class 4 use case".

File: content/install-c4.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+```
+
+The various variables that can be customized are described at <https://github.com/wazo-platform/wazo-ansible/blob/master/README.md#variables>.
+
+By default, Wazo Platform will install the development version. To install
+the latest stable version, activate the following settings in your inventory:
+
+```Ini
+[c4:vars]
+wazo_distribution = wazo-dev-buster
```
Comment (mwolff44) [inline content/install-c4.md]: are you sure that these settings install stable version ?

File: content/install-c4.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+# ansible-galaxy install -r requirements-postgresql.yml
+```
+
+4. Create the Ansible inventory in `inventories/c4` to specify your hosts and your preferences, as follows:
+
+Define one or more SBCs:
+
+```Ini
+[sbc_host]
+sbc_1 ansible_ssh_host=10.0.1.1 ansible_ssh_port=22
```
Comment (mwolff44) [inline content/install-c4.md]: It is better to use IP addresses reserved for documentation : https://tools.ietf.org/html/rfc5737 
blocks 192.0.2.0/24 (TEST-NET-1), 198.51.100.0/24 (TEST-NET-2), and 203.0.113.0/24 (TEST-NET-3)

Comment (tranchitella) [reply content/install-c4.md]: @mwolff44 ok

File: content/install-c4.md (hunk 3 above)
Comment (tranchitella) [reply content/install-c4.md]: @mwolff44 you are right, fixed it.

**Pull request merged**